src/Ouroboros/Network/Protocol/UntypedStream/Client.hs

```diff
@@ -0,0 +1,107 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module Ouroboros.Network.Protocol.UntypedStream.Client where
+
+import Control.Monad (void)
+import Data.Word (Word32)
+import Protocol.Untyped
+
+import Ouroboros.Network.Protocol.UntypedStream.Type
+import Ouroboros.Network.MonadClass
+
+data UntypedStreamClient m rng a t where
+  SendMsgRequest
+    :: rng
```

Comment:
I've still no idea what this is. Random number generator? But what would that be for?

Reply:
I'll change this for `range`, `rng` indeed is a miss-name.